static/js/drafts.js

```diff
@@ -18,6 +18,7 @@ import * as markdown from "./markdown";
 import * as narrow from "./narrow";
 import * as overlays from "./overlays";
 import * as people from "./people";
+import * as rm from "./rendered_markdown";
```

Comment:
In other `*.js` files, this is imported as:
```.js
import * as rendered_markdown from "./rendered_markdown"
```
It's better to stick with the convention.